blkid/blkid_linux_test.go

```diff
@@ -272,7 +361,13 @@ func TestProbePathFilesystems(t *testing.T) {
 						require.NotNil(t, info.Label)
 						assert.Equal(t, test.expectedLabel, *info.Label)
 					} else {
-						assert.Nil(t, info.Label)
+						if test.expectedLabelRegex != "" {
```

Comment:
nit, should be merged into `else if` to avoid nesting, or better refactored to a `switch` statement